src/stores/utilities/pacbioLibraries.js

```diff
@@ -87,8 +87,13 @@ async function validateAndUpdateLibrary(libraryFields) {
   if (!valid.success) {
     return valid
   }
-  const { success, errors } = await updatePacbioLibraryResource(libraryFields)
-  return { success, errors }
+  // const { success, errors, updatedLibrary } = await updatePacbioLibraryResource(libraryFields)
+  const {
+    success,
+    updatedLibrary: { data, included = [] } = {},
```

Comment:
Can we update the tests in tests/unit/stores/utilities/pacbioLibraries.spec.js to also check the behaviour of this updateLibrary field.

Reply:
As I mentioned in the comment above, if we do the destructuring in the updatePacbioLibraryResource method where the response is handled, we can avoid the need for destructuring here.